privval/file.go

```diff
@@ -369,11 +369,6 @@ func (pv *FilePV) signVote(chainID string, vote *cmtproto.Vote, signExtension bo
 	if sameHRS {
 		if bytes.Equal(signBytes, lss.SignBytes) {
 			vote.Signature = lss.Signature
-		} else if timestamp, ok := checkVotesOnlyDifferByTimestamp(lss.SignBytes, signBytes); ok {
-			// Compares the canonicalized votes (i.e. without vote extensions
-			// or vote extension signatures).
-			vote.Timestamp = timestamp
-			vote.Signature = lss.Signature
```

Comment:
This change should stay, and convinces me that we either set all timestamps to 0, or remove the field altogether

Reply:
Sorry, I don't understand why we need to keep it. We assert total equivalence above on line 370.